src/Application/Commands/Business/ImportFromHttpRequest/ImportFromHttpRequestHandler.php

```diff
@@ -93,7 +109,28 @@ public function handle(ImportFromHttpRequestCommand $command)
         }
 
         if ($isCreate) {
-            $this->repository->add($business);
+            $this->businessRepository->add($business);
+        }
+
+        $newPublishingStatus = $business->getPublishingStatus();
+
+        if ($currentPublishingStatus !== PublishingStatus::READY_FOR_REVIEW && $newPublishingStatus === PublishingStatus::READY_FOR_REVIEW
+            && auth()->user()->isEditor()
+        ) {
+            // We have published the business as an Editor.  We want to alert regional admin.  At the moment
+            // editors are not restricted to a region and therefore we alert all of them.
+            $admins = $this->userRepository->findBy([
+                [
+                    'field' => 'repairDirectoryRole',
+                    'operator' => Operators::EQUAL,
+                    'value' => 2
```

Comment:
Hmm, I guess we don't have a constant for the role ids anywhere.  Should we add them to the Role class?

Reply:
I've added some, though some of the code uses a different method - see for example User::isRegionalAdmin.  I've not changed that as part of this, though - let me know if you want me to.

Reply:
No that's OK - thanks for adding it here, more readable now.